redisinsight/api/src/modules/rdi/models/rdi.client.metadata.ts

```diff
@@ -10,6 +10,7 @@ export class RdiClientMetadata {
   sessionMetadata: SessionMetadata;
 
   @IsNotEmpty()
+  @IsOptional()
   @IsString()
-  id: string;
+  id?: string;
```

Comment:
no! id is mandatory field! Beside that we must add validation for required fields inside clients pool/storage